### PR TITLE
Update Download page

### PR DESCRIPTION
**Pull request by patrickdupuis**
This is my attempt at improving the Download page. Feedback and review welcomed!

### CODE REVIEW

File: _includes/download.md [hunk 1]
```diff
@@ -100,40 +117,35 @@
...
 
-<h4>Plugins and Extensions</h4>
+<h2>Plugins and Extensions</h2>
 <ul>
     <li>
-        <a href="https://github.com/supercollider/sc3-plugins">
-            Synthesis Plugins and Extensions
-        </a>
+        <a href="https://github.com/supercollider/sc3-plugins">sc3-plugins</a>
+        <p>Extension plugins for the SuperCollider3 audio synthesis server. These third-party plugins provide additional synthesis, analysis, and other capabilities for the sound server.</p>
```
Comment (telephon) [inline _includes/download.md]: Nowadays, you can write: `Extension plugins for the SuperCollider audio synthesis server.`

Comment (telephon) [reply _includes/download.md]: I don't know if it should be in plural (`scsynth` and `supernova`), but maybe too much information?

Comment (patrickdupuis) [reply _includes/download.md]: I'll keep server singular and change `SuperCollider3` to `SuperCollider`. BTW this text was taken from the sc3-plugins GitHub page. Maybe it should be changed there as well?

**Comment by patrickdupuis** [comment]
@snappizz you ok with these changes? Please have a look when you get the chance :)

**Pull request merged**